code/game/machinery/vending.dm

```diff
@@ -1003,6 +1003,7 @@
 					/obj/item/weapon/gun/projectile/automatic/atreides = 3,
 					/obj/item/weapon/gun/projectile/shotgun/pump/gladstone = 3,
 					/obj/item/weapon/gun/projectile/shotgun/pump = 3,
+					/obj/item/weapon/gun/projectile/automatic/slaught_o_matic = 10,
```

Comment:
Let's increase the amount of pistols in stock to drive home the fact that these things are a dime a dozen. Plus, there's only one FS vendor on the ship.

```suggestion
					/obj/item/weapon/gun/projectile/automatic/slaught_o_matic = 30,
```